guides-markdown/multisig-cli.md

```diff
@@ -0,0 +1,463 @@
+Added key.
+$ bwallet-cli --id=Alice --account=default shared add tpubDDtiGfAVwUAz2wHGh24TCsk6smz8QbHLqr7swEjS8YiEm6BczQPjqRVBuEqPefUCLur1U2VB4P8TUXQg9vXqzQdUx8bynP574sbzuADmRff
+Added key.
+$ bwallet-cli --id=Alice --account=default shared add tpubDDGK1KoY3YQ1xSwz4AyigwWPickr4mFVtRxu7ZZMQz5BDFGNF6SkgJq2kEfTzuudZVgy84rMVZeMv8GDVod2AGxgZSy3fBpWHpYHTEdUkZc
+Added key.
+$ bwallet-cli --id=Alice --account=default shared add tpubDCGKejeBy1oMoZ57NvG2u1X7pffLHkKxew1RAohEzaoPxSSXPdy7st8caWdwymqStLSXxsC4cvKcm9FSLS2PNEcS5rFHo5ccL9xCecqkQth
+Added key.
+```
+
+Now Alice's wallet is fully initialized, generating receive addresses and watching
+the blockchain for transactions!
+
+```
+$ bwallet-cli --id=Alice --account=default account get
+{
+  "name": "default",
+  "initialized": true,
+  "witness": true,
+  "watchOnly": false,
+  "type": "multisig",
+  "m": 3,
+  "n": 5,
+  "accountIndex": 0,
+  "receiveDepth": 1,
+  "changeDepth": 1,
+  "nestedDepth": 1,
+  "lookahead": 10,
+  "receiveAddress": "bcrt1q59779rsw4267cfts6f0q2nhth49casntju7gnf6n52uj75ttzwys3zph8r",
+  "changeAddress": "bcrt1qkefmrhdp2sqgggflfna8uzy72fhj8d6fmg7u3hsft9lvpp0wlmtq7uvkml",
+  "nestedAddress": "2MvGMq49SBDaH7C9wg89aEDZrjBxj9eqTDH",
+  "accountKey": "tpubDDXZKjYarBCHxq9GHQHfRjRVQm8DuVAu33LoigMfHJri1ynqsWVBQbMWQbFU5nT3vabniRrtgJtaDEqnpB9GKQTyac5SbUw4589SvwrYKqQ",
+  "keys": [
+    "tpubDCGKejeBy1oMoZ57NvG2u1X7pffLHkKxew1RAohEzaoPxSSXPdy7st8caWdwymqStLSXxsC4cvKcm9FSLS2PNEcS5rFHo5ccL9xCecqkQth",
+    "tpubDDGK1KoY3YQ1xSwz4AyigwWPickr4mFVtRxu7ZZMQz5BDFGNF6SkgJq2kEfTzuudZVgy84rMVZeMv8GDVod2AGxgZSy3fBpWHpYHTEdUkZc",
+    "tpubDDpd2qFrbXW35YkebSqvihtEsUW9ynmpy61FeVmquPzYzYmFupmabBnHwhJjJighrt2Cmss2Ndg7SJ1iXLuQvZia8KCRWvPTBEacpx6BARf",
+    "tpubDDtiGfAVwUAz2wHGh24TCsk6smz8QbHLqr7swEjS8YiEm6BczQPjqRVBuEqPefUCLur1U2VB4P8TUXQg9vXqzQdUx8bynP574sbzuADmRff"
+  ],
+  "balance": {
+    "tx": 0,
+    "coin": 0,
+    "unconfirmed": 0,
+    "confirmed": 0
+  }
+}
+```
+
+Note that even though the status indicates `"watchOnly": false`, Alice can not spend
+from this wallet by herself. She only has one private key (and four public keys)
+per address.
+
+Bob, Charlie, David, and Erica all go through the same process of importing the other
+keys. In the end, all five participants will have the same wallet structure, and
+**identical** receive addresses. Assuming each person is running their own bcoin
+full node on their own secure machine, they will now have a totally decentralized
+multi-signature wallet! The participants do not need to trust each other to set
+up or use this scheme. Some participants may even be using a hardware wallet like
+a Ledger, connected to their bcoin full node with the
+[bledger](https://github.com/bcoin-org/bledger) package.
+
+## Let's spend some money (as a team!)
+
+So now we have five wallets, each with one unique private key, and four shared
```

Comment:
I think we should use xprv and xpub here. Something to this effect may be more accurate: 

"Each wallet has its own account xprv that can generate many private and public key pairs. The public keys are included in multisig addresses and the private keys are used to add the participant's signature to transactions. Each wallet also has four account xpubs from the other participants. These xpubs can be used to generate the other 4 public keys used in each multisig address. This structure allows each participant to generate the same receiving address while only being able to add their unique signature to transactions."

Reply:
yeah great rewrite, thanks